ts/src/coinsph.ts

```diff
@@ -1720,6 +1740,57 @@ export default class coinsph extends Exchange {
         return this.safeString (statuses, status, status);
     }
 
+    async fetchDepositAddress (code: string, params = {}) {
+        /**
+         * @method
+         * @name coinsph#fetchDepositAddress
+         * @description fetch the deposit address for a currency associated with this account
+         * @see https://coins-docs.github.io/rest-api/#deposit-address-user_data
+         * @param {string} code unified currency code
+         * @param {object} params extra parameters specific to the bitget api endpoint
+         * @param {string} params.network network for fetch deposit address
+         * @returns {object} an [address structure]{@link https://docs.ccxt.com/#/?id=address-structure}
+         */
+        await this.loadMarkets ();
+        const networkCode = this.safeString (params, 'network');
+        const networkId = this.networkCodeToId (networkCode, code);
```

Comment:
@sc0Vu we need to map some networks, for instance 
this works:
`n coinsph fetchDepositAddress "USDT" '{"network":"ETH"}'`
but this does not:
` n coinsph fetchDepositAddress "USDT" '{"network":"ERC20"}'`

Reply:
@carlosmiei turns out, they don't have network like {ERC|TRC|BEP}20 (token spec name). Anyway I push some updates for this. Let me know if you find any issue.

Reply:
@sc0Vu thanks